Source/WebCore/Modules/mediastream/gstreamer/GStreamerWebRTCUtils.cpp

```diff
@@ -619,7 +619,7 @@ GRefPtr<GstCaps> capsFromRtpCapabilities(const RTCRtpCapabilities& capabilities,
             gst_structure_set(codecStructure, "encoding-params", G_TYPE_STRING, makeString(*codec.channels).ascii().data(), nullptr);
 
         if (auto encodingName = gstStructureGetString(codecStructure, "encoding-name"_s)) {
-            if (auto payloadType = payloadTypeForEncodingName(encodingName.toString()))
+            if (auto payloadType = payloadTypeForEncodingName(String(encodingName.span())))
```

Comment:
Why do we have to explicitly write `String()` here? Doesn't it compile without it?